packages/pds/src/api/com/atproto/server/requestPasswordReset.ts

```diff
@@ -19,7 +19,10 @@ export default function (server: Server, ctx: AppContext) {
     handler: async ({ input, req }) => {
       const email = input.body.email.toLowerCase()
 
-      const account = await ctx.accountManager.getAccountByEmail(email)
+      const account = await ctx.accountManager.getAccountByEmail(email, {
+        includeDeactivated: true,
+        includeTakenDown: true,
```

Comment:
Adding support for taken-down accounts to reset their password, update their email.  Just double-checking we like that 👍

Reply:
Removed it from update email, but I do think it's import for resetting your password 👍 

You may want to reset your password in order to export some of your private data or get a signed PLC operation for instance